src/app/routes/utils/regex/utils/index.js

```diff
@@ -96,7 +96,7 @@ export const getTipoHomeRegex = services => {
 };
 
 export const getHomePageRegex = services => {
-  let homePages = services;
+  let homePages = services.filter(service => service !== 'ukchina');
```

Comment:
A bit hacky, but I wanted to ensure that it was not greedy and would use the `getUkChinaHomepageRegex` instead...